scripts/README.md

```diff
@@ -2,62 +2,130 @@ General scripts need for tasks related to the DC2 production and
 validation can be put in this directory.
 
 ### How To Generate Object Tables from DM processing outputs
+The commands run here are from the `DC2-Production/scripts` directory.  The notes in this file using a `SCRIPT_DIR` environment variable to point to this location.  I ran these as I was developing the scripts, so my `SCRIPT_DIR` points to my local checkout:
+
+```
+SCRIPT_DIR=/global/homes/w/wmwv/local/lsst/DC2-production/scripts
+```
+
 1. The Run 1.1p Object table summary files were generated with
 
 ```
 REPO=/global/projecta/projectdirs/lsst/global/in2p3/Run1.1/output
-python merged_tract_cat.py ${REPO} 5066 5065 5064 5063 5062 4852 4851 4850 4849 4848 4640 4639 4638 4637 4636 4433 4432 4431 4430 4429
+python ${SCRIPT_DIR}/merged_tract_cat.py ${REPO} 5066 5065 5064 5063 5062 4852 4851 4850 4849 4848 4640 4639 4638 4637 4636 4433 4432 4431 4430 4429
```

Comment:
There's not an easy and _robust_ way.

For a particular version of the DM Stack there will be a defined structure to filenames used to store the products and one could run a `find` or `glob` through a `cut` and `uniq` and get the list. But in general the list will evolve, but evolve slowly over time.  For Run 2.x it will be different and we can update for that.